docs/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables.md

```diff
@@ -57,7 +57,7 @@ The following restrictions apply to implicitly-typed variable declarations:
 
 - `var` cannot be used on fields at class scope.
 
-- Variables declared by using `var` cannot be used in the initialization expression. In other words, this expression is legal`: int i = (i = 20);` but this expression produces a compile-time error: `var i = (i = 20);`
+- Variables declared by using `var` cannot be used in the initialization expression. In other words, this expression is legal: `int i = (i = 20);` but this expression produces a compile-time error: `var i = (i = 20);`
```

Comment:
I don't know if there is a difference between:

```csharp
int i = (i = 20);
int i = 20;
```

But I think it's worth to mention the difference if there is, or state that they're identical.

Reply:
Maybe someone can create an issue if it's confusing? I just wanted to help fix the markdown.

Reply:
The extra assignment is need to have `i` on the right hand side of an assignment before it's type has been determined. That's why `var` isn't legal in `var = i = (i = 20);`. I'll approve this PR, and :shipit: now.